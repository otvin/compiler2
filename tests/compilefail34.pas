program compilefail34(output);
{cannot sign a boolean constant}

const 
	myfalse = -True;

begin

	if myfalse then
		writeln('myfalse is true');
	else
		writeln('myfalse in false');

end.
