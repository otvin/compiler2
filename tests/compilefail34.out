Cannot have a sign before a Boolean constant in file:tests/compilefail34.pas line:5     column:16 
